api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/security/ComponentsConfiguration.java

```diff
@@ -24,30 +21,4 @@ public BCryptPasswordEncoder passwordEncoder() {
         return new BCryptPasswordEncoder(10);
     }
```

Comment:
Is this class necessary at all now?

Reply:
I think we should keep it, because this class will be moved and changed in catalog-authentication.